swxsoc/util/util.py

```diff
@@ -5,11 +5,31 @@
 import os
 
 from astropy.time import Time
+import astropy.units as u
+import boto3
+from botocore.exceptions import NoCredentialsError
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from parfive import Downloader
```

Comment:
Do we need to add this to the `pyproject.toml` as well? I don't think I saw it there, but I could have missed it.

Reply:
It's implicitly within the sunpy dependency, but I have explicitly set it in the `pyproject.toml`